docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Application Gateway troubleshooting
+type: Troubleshooting
+---
+
+In case user calls registered service and receives an error:
+
+- Verify that the call reached Application Gateway.  
+  To do that fetch logs from Application Gateway pod:
+  ```
+  kubectl -n kyma-integration logs -l app={APP_NAME}-application-gateway -c {APP_NAME}-application-gateway
+  ```
+  If the request reached the pod, it should be logged by Application Gateway.
+  
+  If the call is not in the logs, check if Access Service exists.
+  ```
+  kubectl -n kyma-integration get svc app-{APP_NAME}-{SERVICE_ID}
+  ```
+  If it doesn't, try to deregister the Service using the following command:
```

Comment:
Access Services are created either per service or per registered Application. 

Don't use "the following" unless you really have to. "Run this command to deregister the service" / "to deregister (...) run / run this command:"